lib/checks/aria/unsupportedrole.js

```diff
@@ -1,3 +1 @@
-return !axe.commons.aria.isValidRole(node.getAttribute('role'), {
-	flagUnsupported: true
-});
+return axe.commons.aria.isUnsupportedRole(node.getAttribute('role'));
```

Comment:
I know the `aria.isValidRole` does not `sanitize` the `role`. It may be worth doing `trim` or `axe.commons.text.sanitize`, as well as `toLoweCase`, so it matches the values in the `lookupTable`

Reply:
That's not the bug this is trying to fix, but fair point. Better to use `aria.getRole(node, { noImplicit: true })`, that does everything you need to do to get the role explicit role of an element. It also handles case sensitivity better, and will eventually be able to deal with fallback roles too.

Reply:
Why wouldn't we want the elements implicit role as well?